### PR TITLE
VYOS release 2.0.0 doc update

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +65,10 @@ Name | Description
 [vyos.vyos.vyos_user](https://github.com/ansible-collections/vyos.vyos/blob/main/docs/vyos.vyos.vyos_user_module.rst)|Manage the collection of local users on VyOS device
 [vyos.vyos.vyos_vlan](https://github.com/ansible-collections/vyos.vyos/blob/main/docs/vyos.vyos.vyos_vlan_module.rst)|Manage VLANs on VyOS network devices
 
+### Inventory plugins
```
Comment (NilashishC) [inline README.md:68]: Looks like an empty section, can we remove this?

File: changelogs/CHANGELOG.rst [hunk 2]
```diff
@@ -50,11 +81,6 @@ Minor Changes
 v1.0.3
 ======
 
-Release Summary
```
Comment (NilashishC) [inline changelogs/CHANGELOG.rst:53]: This removes old changelogs?

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -122,9 +122,6 @@ releases:
     - prevent-vyos_config-saving-in-check-mode.yaml
     release_date: '2020-07-31'
   1.0.3:
-    changes:
```
Comment (NilashishC) [inline changelogs/changelog.yaml:125]: Same as above.

File: changelogs/CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,37 @@ Vyos Collection Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Major Changes
```
Comment (NilashishC) [inline changelogs/CHANGELOG.rst:11]: galaxy.yml also needs to be updated

**Review by NilashishC — APPROVED** [review]

**Comment by GomathiselviS** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Pull request merged**